### PR TITLE
Angular Kickoff

### PR DESCRIPTION
**Pull request by francoprud**
## Summary

- Add AngularJS to project.
- Add documentation on how to use it inside `README`  file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,30 @@ Log out from postgres and run:
 
 Your server is ready to run. You can do this by executing `rails server` and going to [http://localhost:3000](http://localhost:3000). Happy coding!
 
+## AngularJS Guide
+
+AngularJS is added inside `vendor/assets/javascripts` folder. The current version is `v1.6.3  `.
```
Comment (epintos) [inline README.md]: You cant include a gem instead?

Comment (francoprud) [inline README.md]: I believe it's better not depending on a gem and just placing inside vendor the desired version of AngularJS.

Comment (francoprud) [inline README.md]: Even though here is the gem for angular: [gem](https://github.com/hiravgandhi/angularjs-rails)

Comment (mishuagopian) [inline README.md]: @epintos IMO, it's better if we can choose the version we want..

File: README.md [hunk 2]
```diff
@@ -72,6 +72,30 @@ Log out from postgres and run:
...
+```ruby
+  - provide(:content_cache, 'no-cache')
+```
+
+This allows to specify to the `meta` tag at the application layout to know if that view must be
+cached or not.
+
+```ruby
+  meta name='turbolinks-cache-control' content="#{yield(:content_cache)}"
+```
```
Comment (mishuagopian) [inline README.md]: I'd clear out this line has been already set for users.

File: app/assets/javascripts/app.module.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+angular.module('rails-bootstrap', []);
+
+/*
+  Instead of including the ng-app to the layout or view it is included on turbolinks:load event.
+  This will ensure the angular application to be properly initialized when using together with
+  Turbolinks.
+*/
+$(document).on('turbolinks:load', function () {
```
Comment (mishuagopian) [inline app/assets/javascripts/app.module.js]: Are we always using this with turbolinks?

File: README.md (hunk 1 above)
Comment (epintos) [inline README.md]: but you can always tell which version you want in the gemfile

Comment (mishuagopian) [inline README.md]: Ok. I read the documentation better and found this:
```
Every attempt is made to mirror the currently shipping Angular.js version number wherever possible.
The major, minor, and patch version numbers will always represent the Angular.js version.
```
So we should fix Gems version to the desired Angular version.

Comment (epintos) [inline README.md]: yeah. I think it can work

File: app/assets/javascripts/app.module.js (hunk 3 above)
Comment (francoprud) [inline app/assets/javascripts/app.module.js]: No, I can leave commented the _not tubolinks way_

File: README.md (hunk 2 above)
Comment (francoprud) [inline README.md]: Sure!

File: README.md (hunk 1 above)
Comment (francoprud) [inline README.md]: Nice, I will set up this to use the gem

**Pull request merged**